### ISSUE TITLE
Documentation Seems to Still be Building From Master Rather than Main

### ISSUE THREAD

**Issue by zm711** [SpikeInterface/spikeinterface#1520]
I was just checking on the documents and noticed that when I click "edit on Github" it goes to `master` instead of `main`.  I started
digging around and in #1501 I made the first changes to `main` instead of master. If we compare what the docs say we see:

![image](https://user-images.githubusercontent.com/92116279/231294706-67d674a5-ade7-41f7-a0df-97c693a1f2f9.png)

Whereas our merged edits present in `main` show the fixes:
![image](https://user-images.githubusercontent.com/92116279/231294816-003afd8c-1ba3-449b-b85a-3cb541348a3c.png)

**Comment by alejoe91** [comment]
You're right! I fixed it  by pointing RTD to `main` instead of `master`! Should be up to date fromt he next build!
